lib/compress/zstd_compress_internal.h

```diff
@@ -1149,11 +1149,12 @@ ZSTD_checkDictValidity(const ZSTD_window_t* window,
 
 MEM_STATIC void ZSTD_window_init(ZSTD_window_t* window) {
     ZSTD_memset(window, 0, sizeof(*window));
-    window->base = (BYTE const*)"";
-    window->dictBase = (BYTE const*)"";
-    window->dictLimit = 1;    /* start from 1, so that 1st position is valid */
-    window->lowLimit = 1;     /* it ensures first and later CCtx usages compress the same */
-    window->nextSrc = window->base + 1;   /* see issue #1241 */
+    window->base = (BYTE const*)" ";
```

Comment:
for my understanding : why `" "` ? avoiding UB pointer arithmetic ?

Reply:
Yeah, CI compiler warnings were complaining about `nextSrc = window->base + 1` being beyond the end of the index.